mapie/tests/test_time_series_regression.py

```diff
@@ -392,3 +414,23 @@ def test_consistent_class() -> None:
     np.testing.assert_allclose(y_pis_1[:, 0, 0], y_pis_2[:, 0, 0])
     np.testing.assert_allclose(y_pis_1[:, 1, 0], y_pis_2[:, 1, 0])
     np.testing.assert_allclose(y_pred_1, y_pred_2)
+
+
+def test_aci_method() -> None:
+    """
+    Test function for the "aci" (Adapted Conformal Inference) method
+    in a MapieTimeSeriesRegressor.
+    Additionally, it attempts to test the regressor with the "enbpi"
+    method, but this part is expected to raise an exception,
+    and it captures the exception without taking any action.
+    """
+    mapie_regressor = MapieTimeSeriesRegressor(method="aci")
+    mapie_regressor.fit(X, y)
+    mapie_regressor.predict(X, alpha=0.05)
+    mapie_regressor.adapt_conformal_inference(X, y)
+    with pytest.raises(AttributeError,
+                       match=r"This method is only Allowed "
+                             r"value is aci method, not for .*"):
```

Comment:
```suggestion
    with pytest.raises(AttributeError,
                       match=r"This method can be called "
                             r"only with method='aci' *"):
```
"This method can be called "
                f"only with method='aci',